src/db_helper.py

```diff
@@ -1,48 +1,53 @@
 from config import db, app
 from sqlalchemy import text
 
-table_name = "todos"
+table_name = 'articles'
 
 def table_exists(name):
   sql_table_existence = text(
-    "SELECT EXISTS ("
-    "  SELECT 1"
-    "  FROM information_schema.tables"
-    f" WHERE table_name = '{name}'"
-    ")"
+    'SELECT EXISTS ('
+    '  SELECT 1'
+    '  FROM information_schema.tables'
+    f' WHERE table_name = \'{name}\''
```

Comment:
No opposition to using backslash, just wondering why it is there?

Reply:
using single quotes in python, so need to escape it in the sql string